_deploy/r/gnoswap/gns/halving.gno

```diff
@@ -22,6 +23,26 @@ import (
 	(365 days) 1 year = 31536000 block
 */
 
+const (
+	HALVING_START_YEAR = int64(1)
+	HALVING_END_YEAR   = int64(12)
+
+	HALVING_AMOUNTS_PER_YEAR = []uint64{
+		18_750_000_000_000 * 12, // Year 1:  225000000000000
+		18_750_000_000_000 * 12, // Year 2:  225000000000000
+		9_375_000_000_000 * 12,  // Year 3:  112500000000000
+		9_375_000_000_000 * 12,  // Year 4:  112500000000000
+		4_687_500_000_000 * 12,  // Year 5:  56250000000000
+		4_687_500_000_000 * 12,  // Year 6:  56250000000000
+		2_343_750_000_000 * 12,  // Year 7:  28125000000000
+		2_343_750_000_000 * 12,  // Year 8:  28125000000000
+		1_171_875_000_000 * 12,  // Year 9:  14062500000000
+		1_171_875_000_000 * 12,  // Year 10: 14062500000000
+		1_171_875_000_000 * 12,  // Year 11: 14062500000000
+		1_171_875_000_000 * 12,  // Year 12: 14062500000000
+	}
```

Comment:
`HALVING_AMOUNTS_PER_YEAR` has a fixed side, I think it would be better to change it to a fixed-size array.

Reply:
https://github.com/gnoswap-labs/gnoswap/blob/b98a424f2714129b7900860392f2f9905753efae/_deploy/r/gnoswap/gns/halving.gno#L30